sdk/eventhub/eventhubs-checkpointstore-blob/samples/downlevelContainerClient.js

```diff
@@ -0,0 +1,42 @@
+/*
+  Copyright (c) Microsoft Corporation. All rights reserved.
+  Licensed under the MIT Licence.
+
+  This sample demonstrates how to extend the Storage Blob Container Client
+  to downlevel the API version used when communicating with the service.
+
+  This may be useful if the environment you are targetting supports an
+  older version of Storage Blob than is officially supported by the
+  @azure/storage-blob SDK.
+*/
+
+const { ContainerClient } = require("@azure/storage-blob");
+
+/**
+ * The DownlevelContainerClient overwrites the API version sent via the
+ * `x-ms-version` header to "2017-11-09".
+ */
+class DownlevelContainerClient extends ContainerClient {
```

Comment:
Please consider renaming the class DownlevelContainterClient to something more generic. For example, the same code could be potentially used for changing the level of the storage API to a newer API.